cmd/push/main.go

```diff
@@ -177,6 +177,7 @@ func mainImpl() error {
 	tags := flag.String("tags", "", "build tags to pass")
 	rel := flag.String("rel", ".", "directory on remote host to push files into")
 	host := flag.String("host", os.Getenv("PUSH_HOST"), "host to push to; defaults to content of environment variable PUSH_HOST")
+	preferredTool := flag.String("tool", "", "tool to push with")
```

Comment:
It's opaque for users which values are valid. Please improve the help string.

Reply:
Done.